linera-core/src/remote_node.rs

```diff
@@ -314,7 +314,7 @@ impl<N: ValidatorNode> RemoteNode<N> {
     }
 
     #[instrument(level = "trace", skip(validators))]
-    async fn download_blob(
+    pub async fn download_blob(
```

Comment:
Let's add a doc comment that this downloads a blob but doesn't verify that it has actually been published and accepted by a quorum of validators.